lib/rubocop/cop/rails/find_by_arg.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # This cop is used to identify usages of `find_by(arg)` and
+      # change them to use `find_by(column: arg)` instead.
+      #
+      # @example
+      #   # bad
+      #   User.find_by(1)
+      #
+      #   # good
+      #   User.find_by(id: 1)
+      class FindByArg < Cop
+        MSG = '`find_by(arg)` may not work.' \
```

Comment:
I'd change this message to something like `Use `find_by(column: arg)` or `find` instead.` or something similar.